packages/hyperproxy-node/src/index.js

```diff
@@ -1,19 +1,31 @@
+import Pino from 'pino';
 import Dat from 'dat-node';
 import Swarm from 'webrtc-swarm';
 import SignalHub from 'signalhubws';
-import HyperproxyHubClient from 'hyperproxy-hub-client';
 import Wrtc from 'wrtc';
 import {w3cwebsocket as WebSocket} from 'websocket';
 
+import HyperproxyHubClient from 'hyperproxy-hub-client';
 import {HUB_MSG_TYPE} from 'hyperproxy-config';
 
+const pretty = Pino.pretty();
```

Comment:
Can you put this in a class and then we can import it and not have this boilerplate?